to_do_list_class.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'to_do_item_class.rb'
+
+class List
+  attr_reader :name
+  #lists know their item names how to view, add, remove, or complete
+  def initialize(name = "New ToDo List")
+    @name = name
+    @list = Hash.new
+  end
+  def [](name)
+    @list[name]
+  end
+
+  def view_list
+    @list.each{|k| puts " #{k}"}
+  end
+
+  def add_item(item)
+    @list[item] = item
```

Comment:
Also, I'm not sure about using item for both the key and value here. If you don't have a meaningful symbol to use, then maybe using an array would make more sense?